src/components/canrawview/gui/crvgui.h

```diff
@@ -50,8 +56,19 @@ struct CRVGui : public CRVGuiInterface {
         ui->tv->horizontalHeader()->setSectionsMovable(true);
         ui->tv->horizontalHeader()->setSortIndicator(0, Qt::AscendingOrder);
         ui->tv->setColumnHidden(0, true);
-        ui->tv->setColumnHidden(1, true);
-        ui->tv->setColumnHidden(3, true);
+
+        tvModel.setHeaderData(
```

Comment:
Use magic numbers + comments here instead of enum. Enum may be misleading as column may be moved to other position